dss/util/version.py

```diff
@@ -6,4 +6,4 @@ def datetime_to_version_format(timestamp: datetime.datetime) -> str:
     return timestamp.strftime(_datetime_format)
 
 def datetime_from_timestamp(ts: str) -> datetime.datetime:
-    return datetime.datetime.strptime(ts, _datetime_format)
+    return datetime.datetime.strptime(ts, _datetime_format).replace(tzinfo=datetime.timezone.utc)
```

Comment:
if this function is used in a lambda, or out in another function, changing the timezone might have an undesired consequence.